pkg/controller/migmigration/backup.go

```diff
@@ -89,6 +97,19 @@ func (t *Task) getInitialBackup() (*velero.Backup, error) {
 	return t.getBackup(labels)
 }
 
+func (t *Task) getUserIncludedResourceList(srcClient compat.Client) ([]string, error) {
+	resources := []string{}
+	includedResources := t.PlanResources.MigPlan.Spec.IncludedResources
+	for _, res := range includedResources {
+		resMapper, err := srcClient.RESTMapper().RESTMapping(schema.GroupKind{Group: res.Group, Kind: res.Kind})
+		if err != nil {
+			return resources, err
```

Comment:
you probably want to aggregate errors here?

One GVK erroring out will stop the processing of all the GVK's below

Reply:
@alaypatel07 This is intentional. Since affects backup, we want to fail as soon as there is an error found.

Reply:
@alaypatel07 as we disussed offline, we will be aggregating errors here so that all GVKs will be processed at once.